edit/edit.css

```diff
@@ -24,6 +24,10 @@ body {
   margin-top: 1rem;
   margin-left: 1.7rem;
 }
+section {
+  box-sizing: border-box;
+  width: 100%;
```

Comment:
This was needed before I added the code (`magicBuffer` variable in `lint.js`) to subtract out a bit of padding to prevent the vertical scrollbar from showing up. Removed now.